### ISSUE TITLE
Unwanted indentation after logical line split

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-testsuitesmanagement#107]
Several times in `RobotFramework_TestsuitesManagement\Config\CConfig.py` I see logical lines splitted into several
physical lines, e.g.:

```
                    logger.error("Additional properties are not allowed! \n \
                    Please put the additional params into 'params': { 'global': {...}")
```
Please keep in mind here that the indentation of the second physical line is part of the resulting string. In most of the cases this is not wanted. In a lot of cases output lines are already long enough. Therefore we do not need additional indentations there.

Either you let all following lines in the code start in the first column or you put all parts of a longer string in separate variables and concatenate these variables to a resulting output string.

**Comment by test-fullautomation** [comment]
Hi Son,
I observed the same issue already.
Please take care of a nice log output - take TML as reference. Bad readability is annoying if you work with logs. This is in general valid. E.g. also multiple colons ```:``` should be always aligned to one column.

In above case you can this to be able to control the indent:
```
 logger.error("Additional properties are not allowed! \n" +  \
              "Please put the additional params into 'params': { 'global': {...}")
```
Thank you,
 Thomas

**Comment by namsonx** [comment]
Hello Thomas and Holger,

I created the pull-request #119  to update the error message format in log.
Now, the format of error message in log is much better :)

Thank you,
Son

**Comment by HolQue** [comment]
I agree. Much better now!

**Comment by test-fullautomation** [comment]
solved and merged with #119
